doc/Language/variables.pod6

```diff
@@ -612,7 +611,8 @@ As with C<my>, declaring multiple C<state> variables must be placed
 in parentheses and for declaring a single variable, parentheses may
 be omitted.
 
-Please note that many operators come with implicit binding, what will lead to actions at a distance. Use C<.clone> or coercion to create a new container that can be bound to.
+Many operators come with implicit binding that would lead to actions at a distance. 
```

Comment:
So:
Many operators come with implicit binding which can lead to actions at a distance.